chemclipse/tests/org.eclipse.chemclipse.msd.model.fragment.test/src/org/eclipse/chemclipse/msd/model/support/CombinedMassSpectrumCalculator_2_Test.java

```diff
@@ -76,14 +76,10 @@ public void testSize_4() {
 	@Test
 	public void testSize_5() {
 
-		/*
-		 * Math round is used to determine the integer value of the mass
-		 * fragment.
-		 */
 		ions.add(new Ion(56.4f, 500.0f));
 		ions.add(new Ion(56.2f, 700.0f));
 		ions.add(new Ion(55.9f, 800.0f));
```

Comment:
Watch out the intensity, when using nominal option, the final intensity shall be 2000.